#pragma once

#include <stdbool.h>

void example_single_number(bool print_encoded);
void example_simple_1(bool print_encoded);
void example_simple_2(bool print_encoded);
void example_simple_nested(bool print_encoded);
void example_simple_repeated(bool print_encoded);
void example_simple_oneof_1(bool print_encoded);
void example_simple_oneof_2(bool print_encoded);
void example_simple_including(bool print_encoded);
